.github/workflows/build.yaml

```
@@ -0,0 +1,22 @@
+name: Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Pull requests to feature and/or release branches should be testable..
```suggestion
  pull_request:
    types: ['opened', 'edited', 'reopened', 'synchronize']
```